packages/bundle-source/src/index.js

```diff
@@ -96,7 +96,9 @@ export default async function bundleSource(
 
     // Parse the rolled-up chunk with Babel.
     // We are prepared for different module systems.
-    const ast = (babelParser.parse || babelParser)(code);
+    const ast = (babelParser.parse || babelParser)(code, {
+      plugins: ['bigInt'],
+    });
```

Comment:
I'm ok with this but I find the need for it surprising. Bigints have been an official part of JS for at least a year now. Are we using an old Babel?